homework27/src/ba/bitcamp/homework27/task01/CountLetter.java

```diff
@@ -0,0 +1,188 @@
+package ba.bitcamp.homework27.task01;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * Ucitati bilo koju tekst datoteku i odrediti koliko se prvo slovo prvog reda
+ * ponavlja u cijelom File-u. Podijeliti posao na vise Thread-ova pomocu
+ * Producer Consumer metodologije, tako da svaki Thread dobije jednu liniju
+ * File-a. Program nije osjetljiv na velika i mala slova.
+ * 
+ * @author amra.sabic
+ *
+ */
+public class CountLetter {
+	// declaration of parameters
+	private static final int WORKERS = 4;
+
+	private static LinkedBlockingQueue<Task> queue;
+	private static ArrayList<Worker> workers;
+	private static Object lock = new Object();
+	private static BufferedReader reader;
+
+	private static int counter = 0;
+
+	/**
+	 * Main
+	 * 
+	 * @param args
+	 */
+	public static void main(String[] args) {
+
+		workers = new ArrayList<>();
+		queue = new LinkedBlockingQueue<Task>();
+
+		File f = null;
+		String line = null;
+		String first = null;
+		String firstLowerCase = null;
+		Task t = null;
+
+		try {
+
+			f = new File("C:/Users/Amra/Documents/workspace/homeworks/homework27/src/lalaland.txt");
+			reader = new BufferedReader(new FileReader(new File(f.getAbsolutePath())));
+
+			// initialize task, add task to queue
+			int num = numberOfLines(f);
```

Comment:
ovdje se bespotrebno citav fajl ucitava da bi se prebrojale linije... to nam ne treba, moze se koristiti while petlja sa uslovom (line=reader.readLine())!=null